Source/Fuse.Elements/Element.HitTest.uno

```diff
@@ -72,9 +72,9 @@ namespace Fuse.Elements
 		{
 			if (HasVisualChildren)
 			{
-				EnsureSortedZOrder();
-				for (int i = ZOrder.Count-1; i >=0; --i)
-					ZOrder[i].HitTest(htc);
+				var zOrder = GetCachedZOrder();
+				for (var i = zOrder.Length; i --> 0; )
```

Comment:
Please don't use this syntax, it's obtuse. Use the normal `for (var i = zOrder.Length-1; i >= 0; --i)`